src/App.js

```diff
@@ -3,19 +3,23 @@ import "./App.css";
 import HomePage from "./components/HomePage";
 import Nav from "./components/Nav";
 import BookPage from "./components/BookPage";
-import ClaimedBookPage from "./components/ClaimedBookPage";
-import AddBook from "./components/Nav/AddBook";
+// import ClaimedBookPage from "./components/ClaimedBookPage";
+import AddBook from "./components/AddBook";
+import { useState } from "react";
 
 function App() {
+  const [claimedUrl, setClaimedUrl] = useState(
+    "https://book-swap-api.dev.io-academy.uk/api/books?claimed=0"
+  );
```

Comment:
Storing the entire URL here is overkill, the only thing that needs to change for each route is 0 and 1, the rest of the URL remains the same. It would be better to go for a simpler solution